### PR TITLE
SDIT-1096 Non-association Reconciliation process

### PR DESCRIPTION
**Pull request by steverendell**
minor tweaks

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/nonassociations/NonAssociationsApiService.kt [hunk 1]
```diff
@@ -34,6 +34,7 @@ class NonAssociationsApiService(private val nonAssociationsApiWebClient: WebClie
       .get()
       .uri {
         it.path("/non-associations")
+          .queryParam("includeClosed", true)
```
Comment (steverendell) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/nonassociations/NonAssociationsApiService.kt:37]: this is no longer the default

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -115,4 +115,4 @@ reports:
       page-size: 20
   non-associations:
     reconciliation:
-      page-size: 20
+      page-size: 200
```
Comment (steverendell) [inline src/main/resources/application.yml:118]: hopefully this will speed up the recon process

**Review by karenmillermoj — APPROVED** [review]
lgtm

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/nonassociations/NonAssociationsApiService.kt (hunk 1 above)
Comment (ushkarev) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/nonassociations/NonAssociationsApiService.kt:37]: 🙇 sorry!

**Pull request merged**